src/QsCompiler/Tests.Compiler/TestCases/Optimizer/Reordering_output.txt

```diff
@@ -0,0 +1,14 @@
+namespace Microsoft.Quantum.Testing.Optimization.Reordering {
```

Comment:
Here, too, having a test where reordering is not done/possible would be good. 
According to what are you reordering? Are you pulling up all "classical" statements to the top such that you can make operation calls be directly after one another? Are you reordering the operation calls as well?